templates/home/register.html

```diff
@@ -21,7 +21,12 @@
                 right: 10px;
                 z-index: 1;
                 color: aliceblue;
-            }
+            } 
+          body{
```

Comment:
Use css to change multiple background images and try to put images in respect to Indian Tourism Places.
Currently you had used a single image.
Always remember to "Fetch Origin" first.